src/components/expanded-state/unique-token/UniqueTokenExpandedStateHeader.tsx

```diff
@@ -91,14 +100,6 @@ const FamilyActions = {
       iconValue: 'safari.fill',
     },
   },
-  [FamilyActionsEnum.etherscan]: {
```

Comment:
Removed this because it's actually dead code. There isn't an `etherscan` property on `FamilyActionsEnum` and there are there are no other references to it in this file. TypeScript pointed this one out 🙏